### ISSUE TITLE
[FIX] Can not generate an API key

### ISSUE THREAD

**Issue by anku94** [khoj-ai/khoj#674]
## Describe the bug

I can not figure out a way to generate an API key with Khoj 1.6.2.

The option is missing from `http://localhost:42110/config#clients`, and I'm not sure if this is the right API, but `"GET /auth/token HTTP/1.1"` is also returning 404 in the logs.

## To Reproduce
Steps to reproduce the behavior: run either the main branch, or the docker image, or the pip package for this version

## Platform
- Server:
    - [ ] Cloud-Hosted (https://app.khoj.dev)
    - [X] Self-Hosted Docker
    - [X] Self-Hosted Python package
    - [X] Self-Hosted source code
- Client:
    - [ ] Obsidian
    - [ ] Emacs
    - [ ] Desktop app
    - [ ] Web browser
    - [ ] WhatsApp
- OS:
    - [ ] Windows
    - [X] macOS
    - [ ] Linux
    - [ ] Android
    - [ ] iOS

### If self-hosted
- Server Version [e.g. 1.0.1]: 1.6.2

**Comment by debanjum** [comment]
Hi @anku94, how did you start Khoj? You do not need to create Khoj API key if the Khoj server running in anonymous mode (i.e started `khoj`with `--anonymous-mode` flag). You can just point the client you want to use (e.g desktop, obsidian, emacs) to the Khoj server URL (usually at http://localhost:42110) and start chatting with it.

Let me know if you're facing some other issues with setting up Khoj? You can use the [setup docs](https://docs.khoj.dev/get-started/setup/#3-configure) to self-host Khoj

**Comment by anku94** [comment]
Sorry for the delayed response.

I looked up my Obsidian devtools console, and looks like my problem was being caused by a malformed url (`localhost:42110`, instead of `http://localhost:42110`, the former results in a network request to `app://obsidian.md/localhost:42110/api/...` - which is invalid). 

I understand that this project is in its early stages, and the million ways in which users break things are not handled gracefully yet. Closing the issue for now!
